src/components/svg/networkDrawing.tsx

```diff
@@ -194,6 +198,92 @@ export const arcEdgeGenerator = (size) => {
   }
 }
 
+const ArrowedPath = (props) => {
+  const { d, width } = props
+  const pathRef = useRef(null)
+  useEffect(() => {
+    if (pathRef?.current) {
+      const arrowHeadSize = Math.max(width / 5, 2)
+      let arrows = pathArrows()
+        .arrowLength(arrowHeadSize * 2.5)
+        .gapLength(100)
+        .arrowHeadSize(arrowHeadSize)
+        .path(() => d)
+
+      select(pathRef.current).call(arrows)
+
+      select(pathRef.current)
+        .selectAll(":not(.arrow-head)")
+        .style("fill", "none")
+        .style("stroke-width", arrowHeadSize / 4)
+        .style("stroke", "white")
+
+      select(pathRef.current).selectAll(".arrow-head").style("fill", "white")
+    }
+  }, [!!pathRef?.current])
```

Comment:
Refs don't trigger re-renders, so this will not have any effect

Reply:
Okay take a look now I think I got a better parameter (and also noticed that I need to clean up the arrows on a re-render)